### ISSUE TITLE
SetUID attack vector space

### ISSUE THREAD

**Issue by saurav2658** [SANTHOSH17-DOT/attack-vectors-suite#10]
I would like to contribute to this repo by explaining about attack mentioned above. Please assign me this issue under hacktober fest 2022.
Thank You!

**Comment by Bharadwajshivam28** [comment]
@SANTHOSH17-DOT  

Please assign this to me I would like to work on it

**Comment by Bharadwajshivam28** [comment]
@SANTHOSH17-DOT can i add imaged related to this attack

**Comment by SANTHOSH17-DOT** [comment]
Sure you can add images

**Comment by Bharadwajshivam28** [comment]
code by me will be written in code of conduct? @SANTHOSH17-DOT

**Comment by SANTHOSH17-DOT** [comment]
No. Create a new directory for this attack and add your work there.

**Comment by Bharadwajshivam28** [comment]
ok @SANTHOSH17-DOT

**Comment by Bharadwajshivam28** [comment]
@SANTHOSH17-DOT  Should i make changes in new branch ? Like named setuid branch

**Comment by SANTHOSH17-DOT** [comment]
Name of the branch isn't a problem. Make sure that the directory name is "SET UID"

**Comment by Bharadwajshivam28** [comment]
@SANTHOSH17-DOT  ok fine , so changes made by me will be pushed in to the main  branch

**Comment by Bharadwajshivam28** [comment]
like there is no need to create another branch  but the directory name should be SETUID

**Comment by Bharadwajshivam28** [comment]
and is there also need to create markdown file ? @SANTHOSH17-DOT

**Comment by SANTHOSH17-DOT** [comment]
Yes to describe the research you've made

**Comment by Bharadwajshivam28** [comment]
@SANTHOSH17-DOT  ok but i  am asking that is it necessary to create another branch  or should go with the same branch

**Comment by SANTHOSH17-DOT** [comment]
You can do anything. But avoid merge conflicts when you raise a PR. I would suggest you to create a new branch

**Comment by Bharadwajshivam28** [comment]
ok @SANTHOSH17-DOT

**Comment by Akanksha038** [comment]
I would like to contribute to this project. If you think that would be helpful kindly assign this issue to me. Thank You!
